ggu/main.c

```diff
@@ -330,12 +330,12 @@ process_single_packet(struct rte_mbuf *pkt, const struct ggu_config *ggu_conf)
 
 /* Information needed to submit GGU packets to the GGU block. */
 struct ggu_request {
-	/* GT-GK Unit packets. */
-	struct rte_mbuf *pkts[GATEKEEPER_MAX_PKT_BURST];
-
 	/* Number of packets stored in @pkts. */
 	unsigned int    num_pkts;
-};
+
+	/* GT-GK Unit packets. */
+	struct rte_mbuf *pkts[0];
+} __attribute__((packed));
```

Comment:
Why did you add `__attribute__((packed))` to `struct ggu_request`? I think we shouldn't use it here.